generate-templates.yaml

```diff
@@ -37,11 +37,24 @@
       osinfoname: "{{ oslabels[0] }}"
       cloudusername: cloud-user
 
-
   - name: Load RHEL 8 versions
     set_fact:
       rhel8_labels: "{{ lookup('osinfo', 'distro=rhel') |map(attribute='short_id') |select('match', '^rhel8\\.') |list |sort }}"
 
+  - name: Generate RHEL 8 SAP HANA template
+    template:
+      src: saphana.tpl.yaml
+      dest: "{{ playbook_dir }}/dist/templates/{{ os }}-{{ item.workload }}-{{ item.flavor }}.yaml"
+    with_items:
+      - {flavor: tiny,  workload: saphana, memsize: "24Gi",   cpus: 4, iothreads: True,  emulatorthread: True,  tablet: False, default: False}
```

Comment:
You don't have to specify `memsize` and `cpus` here, since you specify them as template parameters

Reply:
Done.